.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -262,13 +256,41 @@ jobs:
 
       - name: Provision K8s on LVH VM
         if: ${{ steps.vars.outputs.downgrade_version != '' }}
+        id: lvh-kind
         uses: ./.github/actions/lvh-kind
         with:
           test-name: e2e-conformance
           kernel: ${{ matrix.kernel }}
           kind-params: "${{ steps.kind-params.outputs.params }}"
           kind-image: ${{ env.KIND_K8S_IMAGE }}
 
+      - name: Set up job variables for connectivity tests
+        id: vars-conn
+        run: |
+          EXTRA_CLI_FLAGS=(
```

Comment:
This set of flags is the same in the three workflows where you add it, right? Would it be possible to set them and export them from the `lvh-kind` action, to avoid repeating this chunk, maybe?